playground/backend/internal/fs_tool/fs_test.go

```diff
@@ -30,18 +30,29 @@ const (
 	destinationDir  = "destinationDir"
 	testFileMode    = 0755
 	pipelinesFolder = "executable_files"
+	fileName        = "file.txt"
+	directoryName   = "incorrect"
 )
 
+func TestMain(m *testing.M) {
+	err := prepareFiles()
+	if err != nil {
+		panic(fmt.Errorf("error during test setup: %s", err.Error()))
+	}
+	defer teardownFiles()
+	m.Run()
+}
+
 func prepareFiles() error {
-	err := os.Mkdir(sourceDir, testFileMode)
+	err := os.MkdirAll(filepath.Join(sourceDir, directoryName), testFileMode)
```

Comment:
Do we prepare files in the directory with the name "incorrect"? This might be confusing, can we think about a better naming?

Reply:
Done